src/routes/preHandlers/courses.js

```diff
@@ -106,7 +106,8 @@ exports.getCourseTrainee = async (req) => {
       const traineeAlreadyRegistered = course.trainees.some(t => t.toHexString() === trainee._id.toHexString());
       if (traineeAlreadyRegistered) throw Boom.conflict(translate[language].courseTraineeAlreadyExists);
     } else {
-      const missingFields = !payload.company || !get(payload, 'local.email') || !get(payload, 'identity.lastname');
+      const missingFields = !payload.company || !get(payload, 'local.email') || !get(payload, 'identity.lastname') ||
```

Comment:
es tu vraiment obligé de garder le get dans une condition ?

Reply:
(ou alors il y a risque que local ou identity n'apparraissent pas il va dire cannot access property email of undefined ?)

Reply:
oui, il va renvoyer une erreur et pas false